moon-store-mac-os/moon-store-mac-os/Views/User/UserListViewModel.swift

```diff
@@ -101,4 +103,9 @@ final class UserListViewModel: ObservableObject {
             }
         }
     }
+    
+    private func avoidShowLoggedUser() {
```

Comment:
No esta mostrando nada, esta filtrando los usuarios, el nombre es engañoso.

```suggestion
    private func filterUsersSkippingLoggerUser() {
```